tests/roles/ceph_migrate/tasks/mon.yaml

```diff
@@ -126,35 +126,39 @@
         msg: "{{ mon_ipaddr }}"
 
     - name: MON - Patch os-net-config config and REMOVE the current mon IP address
-      delegate_to: "{{ cur_mon.split('.')[0] }}.ctlplane"
+      delegate_to: "{{ cur_mon }}"
       ansible.builtin.lineinfile:
         path: "{{ os_net_conf_path }}"
         state: absent
         regexp: '{{ mon_ipaddr }}/24'
         backup: true
+      vars:
+        os_net_conf_path: "/etc/os-net-config/tripleo_config.yaml"
 
     - name: MON - Refresh os-net-config
-      delegate_to: "{{ cur_mon.split('.')[0] }}.ctlplane"
+      delegate_to: "{{ cur_mon }}"
       ansible.builtin.shell:
         "os-net-config -c {{ os_net_conf_path }}"
+      vars:
+        os_net_conf_path: "/etc/os-net-config/tripleo_config.yaml"
```

Comment:
controller nodes have `tripleo_config` instead of just `config.yaml` that we can find in the EDPM nodes.